src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -234,78 +233,22 @@ namespace openstudio{
       close();
     }
 
-    struct PreparedStatement
+    void SqlFile_Impl::execAndThrowOnError(const std::string &t_stmt)
```

Comment:
execAndThrowOnError is different, relies on sqlite3_exec to execute several statement, so no binding parameters possible